oaknational/Oak-Web-Application
# fix: maths ks4 curriculum map download - LESQ-332

## Description

- Fixes ks4, maths, legacy curriculum map download (zip on programme page)
- The zip file is downloaded and the user remains on the page
- refactored the error message so it works for both buttons.

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1941--oak-web-application.netlify.thenational.academy/teachers/key-stages/ks4/subjects/maths-l/programmes
(should download a zip)
check other downloads are working

## Checklist
